.pre-commit-hooks.yaml

```diff
@@ -14,17 +14,48 @@
     exclude: \.+.terraform\/.*$
     require_serial: true
 
+-   id: checkov_container
+    name: Checkov
+    description: This hook runs checkov.
+    entry: bridgecrew/checkov:latest -d .
```

Comment:
we usually recommend to run docker with `--tty` for better output handle.

Reply:
For reference, this is how `pre-commit` runs [docker_image](https://pre-commit.com/#docker_image) hooks:

https://github.com/pre-commit/pre-commit/blob/bde292b51078357384602ebe6b9b27b1906987e5/pre_commit/languages/docker.py#L111-L146

I pushed up a commit to add `--tty` and the output looks like this now:

```shell
❯ pre-commit run -av checkov_secrets_container
Checkov Secrets..........................................................Passed
- hook id: checkov_secrets_container
- duration: 17.04s

secrets scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 49




❯ pre-commit run -av checkov_secrets_container
Checkov Secrets..........................................................Failed
- hook id: checkov_secrets_container
- duration: 17.16s
- exit code: 1

secrets scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 49

Check: CKV_SECRET_6: "Base64 High Entropy String"
	FAILED for resource: a2478cff5623ee9e46dfe2ab06e96bfe5168f732
	File: /foo/bar-values.yaml:7-8
	Guide: https://docs.paloaltonetworks.com/content/techdocs/en_US/prisma/prisma-cloud/prisma-cloud-code-security-policy-reference/secrets-policies/secrets-policy-index/git-secrets-6.html

		7 | superSecret: c2Vjcm******************************************************************************************************************************************************************************




❯ pre-commit run -a checkov_secrets_container
Checkov Secrets..........................................................Failed
- hook id: checkov_secrets_container
- exit code: 1

secrets scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 49

Check: CKV_SECRET_6: "Base64 High Entropy String"
	FAILED for resource: a2478cff5623ee9e46dfe2ab06e96bfe5168f732
	File: /foo/bar-values.yaml:7-8
	Guide: https://docs.paloaltonetworks.com/content/techdocs/en_US/prisma/prisma-cloud/prisma-cloud-code-security-policy-reference/secrets-policies/secrets-policy-index/git-secrets-6.html

		7 | superSecret: c2Vjcm******************************************************************************************************************************************************************************




❯ pre-commit run -a checkov_secrets_container
Checkov Secrets..........................................................Passed


```